webui/src/app/app.component.html

```diff
@@ -26,7 +26,12 @@
     <mat-menu #userMenu="matMenu">
       <button mat-menu-item (click)="signOut()">Sign out</button>
     </mat-menu>
-    <img class="avatar" [src]="authProcess?.getUserPhotoUrl()" [matMenuTriggerFor]="userMenu"/>
```

Comment:
I think it would be better to move the `[matMenuTriggerFor]="userMenu"` out to a common container, because the if only different the representation, not the function. 

So something like
```
<div [matMenuTriggerFor]="userMenu">
    <span *ngIf="avatarImageUrl; else defaultAvatar">
      <img class="avatar" [src]="avatarImageUrl"/>
    </span>
    <ng-template #defaultAvatar>
      <mat-icon class="avatar">account_circle</mat-icon>
    </ng-template>
</div>
```
Well if that doesn't make styling more difficult.

Reply:
Done, PTAL.